test/test_quantization.py

```diff
@@ -1081,6 +1081,43 @@ def test_skip_quant(self):
                 inplace=False)
             self.assertEqual(model_quantized(self.calib_data[0][0]), result_eager)
 
+    def test_single_linear_dynamic(self):
+        r"""Compare the result of dynamic quantization of single linear layer in
+        eager mode and graph mode.
+        """
+        # eager mode
+        annotated_linear_model = AnnotatedSingleLayerLinearModel().eval()
+        linear_model = SingleLayerLinearModel().eval()
+        # copy the weight from eager mode so that we can
+        # compare the result of the two quantized models later
+        linear_model.fc1.weight = torch.nn.Parameter(annotated_linear_model.fc1.module.weight.detach())
+        linear_model.fc1.bias = torch.nn.Parameter(annotated_linear_model.fc1.module.bias.detach())
+        qconfig_dict = {'': default_dynamic_qconfig}
+        model_eager = quantize_dynamic(annotated_linear_model, qconfig_dict)
+
+        model_traced = torch.jit.trace(linear_model, self.calib_data[0][0])
+        model_script = torch.jit.script(linear_model)
+        result_eager = model_eager(self.calib_data[0][0])
+
+        for model_under_test in [model_traced, model_script]:
+            model_quantized = quantize_dynamic_script(
+                model_under_test,
+                qconfig_dict,
+                test_only_eval_fn,
+                [self.calib_data])
+            self.assertEqual(model_quantized(self.calib_data[0][0]), result_eager)
+
+            # Check to make sure choose_qparams->quant->dequant->linear is numerically
+            # equivalent to the final quantized model.
+            model_fake_quantized = quantize_dynamic_script(
```

Comment:
you mean model_dynamic_quantized?

Reply:
Oh, this is debug, could you use another name? fake_quantize has a special meaning in quantization

Reply:
No, this tests the fake quantized model with the `quantize_per_tensor` op followed by float Linear op to be numerically equivalent to the dynamically quantized op in eager mode.